assets/app/routes.js

```diff
@@ -15,18 +15,20 @@ const Stub = () =>
 
 export default (
   <Route path="/" component={App}>
-    <IndexRedirect to="sites" />
+    <IndexRedirect to="sites"/>
     <Route path="sites">
       <IndexRoute component={Dashboard}/>
       <Route path=":id" component={SiteContainer}>
-        <IndexRoute component={PagesContainer}/>
+        <IndexRoute component={SitePagesContainer}/>
+        <Route path="tree/:fileName" component={SitePagesContainer}/>
```

Comment:
I love this distinction between the tree and actual edit routes, much clearer than before!